ui/components/app/assets/asset-list/network-filter/network-filter.tsx

```diff
@@ -31,10 +31,15 @@ import {
 import UserPreferencedCurrencyDisplay from '../../../user-preferenced-currency-display';
 import {
   CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP,
-  TEST_CHAINS,
+  FEATURED_NETWORK_CHAIN_IDS,
+  FEATURED_NETWORK_NAMES,
+  FEATURED_RPCS,
+  MAINNET_DISPLAY_NAME,
 } from '../../../../../../shared/constants/network';
 import { useGetFormattedTokensPerChain } from '../../../../../hooks/useGetFormattedTokensPerChain';
 import { useAccountTotalCrossChainFiatBalance } from '../../../../../hooks/useAccountTotalCrossChainFiatBalance';
+import InfoTooltip from '../../../../ui/info-tooltip';
+import { CHAIN_IDS } from '@metamask/transaction-controller';
```

Comment:
`shared/constants/network.ts` would be a better place to import from

Reply:
Woops, this was an automatic import. Thanks for catching.